docs/source/development/controllers.rst

```diff
@@ -0,0 +1,221 @@
+Developing a new controller
+===========================
+
+In eVolver controllers are the components that implement experiment logic. They
+can read the state and history of the eVolver environment (e.g. the sensor
+values) and subsequently modify it (via effectors). In a typical eVolver setup
+controller logic is executed once during a device loop, after reading the state
+of all sensors. See :doc:`/concepts` for more details on the device
+operation.
+
+Controllers are implemented as Python classes that inherit from the
+:class:`evolver.controller.Controller` class, the logic of which is executed via
+a call to the :meth:`evolver.controller.Controller.control` method. Developing a
+new controller is a matter of implementing this method. Within
+:meth:`evolver.controller.Controller.control`, code can read sensor values (e.g.
```

Comment:
I wouldn't use "read" here as it implies that subsequent calls to ``get`` have scope for values to vary, which they don't as the "read" happens in the loop and the "get" retrieves these stashed values.